src/cc65/codeinfo.c

```diff
@@ -850,6 +853,63 @@ cmp_t FindTosCmpCond (const char* Name)
 
 
 
+const char* GetCmpSuffix (cmp_t Cond)
+/* Return the compare suffix by the given a compare condition or 0 on failure */
+{
+    /* Check for the correct subroutine name */
+    if (Cond >= 0       &&
+        Cond != CMP_INV &&
+        (unsigned)Cond < sizeof (CmpSuffixTab) / sizeof (CmpSuffixTab[0])) {
+        return CmpSuffixTab [Cond];
```

Comment:
Style point: no space in front of the `[`.

Reply:
Fixed.